tests/integration-tests/conftest.py

```diff
@@ -1382,6 +1418,133 @@ def placement_group_stack(cfn_stacks_factory, request, region):
         logging.warning("Skipping deletion of CFN stacks because --no-delete option is set")
 
 
+@pytest.fixture()
+@pytest.mark.usefixtures("setup_credentials")
+def odcr_stack_factory(request, region, cfn_stacks_factory, vpc_stack):
```

Comment:
I would agree, however, in this case I think the resources need to be generated dynamically based on the cluster configuration? But maybe there is a better way to do this through a template, I'm not sure.

Reply:
I think I now see what you're getting at - would it make more sense to dynamically create a template and then create a stack for the ODCR? This way, if the testing errored out before it could cleanup for some reason, the resources could be reclaimed through deleting the stack rather than digging through various AWS consoles trying to track them down. Wdyt?